packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -293,6 +297,19 @@ abstract class WidgetsBinding extends BindingBase with SchedulerBinding, Gesture
       }
     );
 
+    assert(() {
+      // Expose the ability to send Widget rebuilds to the as [Timeline] events.
+      registerBoolServiceExtension(
+        name: 'profileWidgetBuilds',
+        getter: () => Future<bool>.value(debugProfileBuildsEnabled),
```

Comment:
any reason this isn't just
getter: () async => debugProfileBuildsEnabled,

?

Reply:
No reason - I'll simplify this.